shopelectro/views/search.py

```diff
@@ -26,7 +26,7 @@ def get_redirect_search_entity(self):
         ),
         search_engine.Search(
             name='page',
-            qs=ExcludedModelTPage.objects.get_active(),
+            qs=ExcludedModelTPage.actives.all(),
```

Comment:
We have to use here `ExcludedModelTPage.objects.active()`, because we override `objects` [here](https://github.com/fidals/shopelectro/blob/master/shopelectro/models.py#L222)